--- esphome/platformio.ini	2023-05-28 22:25:22.000000000 +0000
+++ platformio.ini	2023-06-06 18:43:51.265191717 +0000
@@ -4,14 +4,15 @@
 ; It's *not* used during runtime.
 
 [platformio]
-default_envs = esp8266-arduino, esp32-arduino, esp32-idf
+#default_envs = esp8266-arduino, esp32-arduino, esp32-idf
+default_envs = esp32-arduino
 ; Ideally, we want src_dir to be the root directory of the repository, to mimic the runtime build
 ; environment as best as possible. Unfortunately, the ESP-IDF toolchain really doesn't like this
 ; being the root directory. Instead, set esphome/ as the source directory, all our sources are in
 ; there anyway. Set the root directory as the include_dir, so that the esphome/ directory is on the
 ; include path.
 src_dir = esphome
-include_dir = .
+include_dir = esphome
 
 ; This are just the build flags as set by the runtime.
 [flags:runtime]
